client/src/components/ListGroup/ListGroup.scss

```diff
@@ -55,11 +56,16 @@
 
 .list-group-item__thumbnail {
   width: 64px;
-  height: 64px;
+  height: auto;
   display: block;
   background: $gray-lighter;
   float: left;
-  margin: -12px 12px 0 #{0 - $grid-gutter-width / 2};
+  margin: -12px 12px -12px #{0 - $grid-gutter-width / 2};
+
+  > img {
+    max-height: 65px;
+    height: 100%;
```

Comment:
![silverstripe_-_files](https://cloud.githubusercontent.com/assets/318116/25112980/b138b5e2-2448-11e7-85ec-2665c0d3358c.png)

True, I don't mind not stretching the image so they're consistent everywhere but this is what I see in the AssetAdmin at the moment. Pretty sure the last image is vertically stretched or is it a bug?